packages/teleport/src/config.ts

```diff
@@ -118,11 +120,13 @@ const cfg = {
   },
 
   getAuthProviders() {
-    return cfg.auth && cfg.auth.providers ? cfg.auth.providers : [];
+    return cfg.auth && cfg.auth.providers
+      ? (cfg.auth.providers as AuthProvider[])
```

Comment:
instead of casting in every place, could you do the following instead in the config.ts?

```ts
  auth: {
    localAuthEnabled: true,
    providers: [] as AuthProvider[],
    second_factor: 'off' as Auth2faType,
    authType: 'local',
  },
```

Reply:
revised